analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.CSharp8.cs

```diff
@@ -392,17 +392,17 @@ void Method()
                 continue;
             }
 
-            if (sb is null)     // Noncompliant FP
+            if (sb is null)             // Noncompliant FP
             {
                 sb = new StringBuilder();
             }
 
             sb.Append(i);
         }
 
-        if (sb is null) // FN
+        if (sb is null)                 // Noncompliant
```

Comment:
This must be FP by now. sb is never null

Reply:
sb is never null => TP
or am I understanding you wrong?

Reply:
Yes, you're right